### PR TITLE
Update portMappings to include containerPortRange

### PR DESCRIPTION
**Pull request by asmithdt**
## what

Allow specifying containerPortRange in portMappings

## why

To support containerPortRange

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -145,11 +146,12 @@ variable "container_definition" {
...
-    name          = optional(string)
-    appProtocol   = optional(string)
+    containerPort      = optional(number)
+    hostPort           = optional(number)
+    protocol           = optional(string)
+    name               = optional(string)
+    appProtocol        = optional(string)
+    containerPortRange = optional(string)
   }))
   description = "The port mappings to configure for the container. This is a list of maps. Each map should contain \"containerPort\", \"hostPort\", and \"protocol\", where \"protocol\" is one of \"tcp\" or \"udp\". If using containers in a task with the awsvpc or host network mode, the hostPort can either be left blank or set to the same value as the containerPort"
```
Comment (joe-niland) [inline variables.tf:156]: It would be good to include `containerPortRange` in the description if possible

**Comment by joe-niland** [comment]
Hi @asmithdt 
The tests will need updating to allow the Terratest check to pass.
Are you happy to take a look?

**Comment by YeomansIII** [comment]
@asmithdt Anything I can do to help on this? Honestly not sure how to get `containerPortRange` to work in terraform at all, even though it supported on ECS. If anyone has any workarounds in the meantime, like having terraform forgo any validation of the container_definition (I can't think of another way to get containerPortRange into AWS...)

**Comment by joe-niland** [comment]
Hi @asmithdt 

Sorry for the delay. 

Could you please run the following and commit the result?

```
make init
make readme
```

**Pull request closed without merging**